tests/handlers/test_home_handler.py

```diff
@@ -46,13 +29,13 @@ def coro(*args, **kwargs):
     return coro
 
 
-class TestHomeHandler(TempMixin, AsyncHTTPTestCase):
+class TestHomeHandler(TempMixin, utils.AsyncHTTPTestCase):
     def setUp(self):
         self._old_proxy_api_token = os.environ.get("PROXY_API_TOKEN", None)
         os.environ["PROXY_API_TOKEN"] = "dummy_token"
 
         self._bind_unused_port_orig = tornado.testing.bind_unused_port
-        tornado.testing.bind_unused_port = _bind_unused_port
+        tornado.testing.bind_unused_port = utils.bind_unused_port
```

Comment:
utils.AsyncHTTPTestCase.setUp has this setup already so this is not needed (and line 46 as well)